en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+
+    def apply()(implicit session: DBSession): Boolean
+
+By the way, you can name the implicit parameter freely as long as it is unique in its scope, so there is no problem to write it simply as `implicit s =>`. From this point in this book, I may also write as `{implicit s =>}`.
+
+
+### Executing DML
+
+If no exception occured in the previous `create table`, a table should have been created already. Let's issue a `select` statement to the `members` table.
+
+    val members: List[Map[String, Any]] = DB readOnly { implicit session =>
+      SQL("select * from members").map(rs => rs.toMap).list.apply()
+    }
+    // => members: List[Map[String,Any]] = List()
+
+An empty `List` have been returned because there are still no data.
+
+Now, let's try to insert some records. Since the part that starts with `SQL` does not actually issue the SQL until you call the `apply()`, you can re-use it as many times as you like, such as;
+
+    import org.joda.time._
+    DB localTx { implicit session =>
+      val insertSql = SQL("insert into members (name, birthday, created_at) values (?, ?, ?)")
+      val createdAt = DateTime.now
+    
+      insertSql.bind("Alice", Option(new LocalDate("1980-01-01")), createdAt).update.apply()
+      insertSql.bind("Bob", None, createdAt).update.apply()
+    }
+
+
+By the way, ScalikeJDBC allows you to use not only the normal JDBC template shown above, but also the named SQL template where you embed binding variables as `{name}`;
+
+    SQL("insert into members (name, birthday, created_at) values ({name}, {birthday}, {createdAt})")
+      .bindByName('name -> name, 'birthday -> None, 'createdAt -> createdAt)
+      .update.apply()
+
+as well as the executable template where you write binding variables in the comments accompanied by dummy values;
+
+    SQL("""
+      insert into members (name, birthday, created_at) values (
+        /*'name*/'Alice', 
+        /*'birthday*/'1980-01-01', 
+        /*'createdAt*/current_timestamp
+      )
+      """)
+      .bindByName('name -> name, 'birthday -> None, 'createdAt -> createdAt)
+      .update.apply()
+
+This is explained in more detail at the SQL template section in detail.
+
+Let's return to the example and issue the same `select` statement once again.
+
+    val members: List[Map[String, Any]] = DB readOnly { implicit session =>
+      SQL("select * from members").map(_.toMap).list.apply()
+    }
+    // => members: List[Map[String,Any]] = List(Map(ID -> 1, NAME -> Alice, BIRTHDAY -> 1980-01-01, CREATED_AT -> 2012-12-31 00:02:09.247), Map(ID -> 2, NAME -> Bob, CREATED_AT -> 2012-12-31 00:02:09.247))
+
+The two records that was inserted has been returned as expected. You find that the previous `insert` was processed successfully.
+
+Previously in the `select` examples, we obtained our results as `Map[String, Any]`, but let's change it so that the results are mapped to a `Member` class.
```

Comment:
In the paste `select` query examples, we obtained results as `Map[String, Any]` values. Let's rewrite them to map results to a class named `Member` now.